README.rst

```diff
@@ -66,6 +42,26 @@ Why use coala?
   without you having to worry.
 - *Unified interface*: One tool for all programming languages.
```

Comment:
BTW I think bold would look better compared to italics. (It kind of removes the monotonity of the text.)

But that's upto you.